_includes/author-profile.html

```diff
@@ -62,6 +62,9 @@ <h3 class="author__name">{{ author.name }}</h3>
       {% if author.scopus %}
         <li><a href="{{ author.scopus }}"><i class="ai ai-scopus icon-pad-right"></i>Scopus</a></li>
       {% endif %}
+      {% if author.academia %}
+        <li><a href="{{ author.academia }}"><i class="fab fa-fw fa-academia icon-pad-right" aria-hidden="true"></i>Academia</a></li>
+      {% endif %}
```

Comment:
Should be alphabetically listed with the rest of the social media that uses Font Awesome icons.

Reply:
Depending upon what the rendering issue is, this might be a simple fix. The same note applies to the section, but it looks like you want to be using the Academicons instead of Font Awesome - https://jpswalsh.github.io/academicons/